test/include-test.bats

```diff
@@ -3,7 +3,7 @@
 @test "should not import if it's already defined" {
   __bp_imported="defined"
   source "${BATS_TEST_DIRNAME}/../bash-preexec.sh"
-  [ -z $(type -t __bp_preexec_and_precmd_install) ]
```

Comment:
This bug makes test always pass, leading to false negative. For example, changing `__bp_imported` to `__bp_imported_foobar` should have failed the test, but it doesn’t.

Reply:
Ah man, looks like this was broken in 4744d7e6fd105e. Fixed the function below then, but missed this one :/ Good fix 👍